api/src/main/java/org/pmiops/workbench/initialcredits/InitialCreditsExpirationServiceImpl.java

```diff
@@ -17,4 +67,64 @@ public Optional<Timestamp> getCreditsExpiration(DbUser user) {
         // .filter(() -> institutionService.shouldBypassForCreditsExpiration(user))
         .map(DbUserInitialCreditsExpiration::getExpirationTime);
   }
+
+  private void checkExpiration(DbUser user, Timestamp now) {
```

Comment:
Why you pass `now` to this function? For easier unit test?  Or that timestamp is used in upper class? In general this should be a local variable if possible

Reply:
I am passing now to the function, because if not, each user's expiration would be compared to a slightly different now. The first user and the last user might be off by a second or two. This way ensures that all users are being compared to the same timestamp.